src/libraries/System.Transactions.Local/src/System/Transactions/TransactionManager.cs

```diff
@@ -325,7 +350,7 @@ public static TimeSpan MaximumTimeout
                     etwLog.MethodEnter(TraceSourceType.TraceSourceBase, "TransactionManager.get_DefaultMaximumTimeout");
                 }
 
-                LazyInitializer.EnsureInitialized(ref s_maximumTimeout, ref s_cachedMaxTimeout, ref s_classSyncObject, () => DefaultSettingsSection.Timeout);
```

Comment:
This looks like a bug when initializing MaximumTimeout.

Reply:
Yes that was a bug. Good catch. Although it didn't matter previously as no values could be updated before.